### PR TITLE
uv no longer requires editable installs

### PR DESCRIPTION
**Pull request by carmocca**
Reacting to https://github.com/Lightning-AI/litgpt/pull/952#issuecomment-2016972782

### CODE REVIEW

**Comment by carmocca** [comment]
Blocked by https://github.com/Lightning-AI/litgpt/pull/952#issuecomment-2017075104

**Comment by Andrei-Aksionov** [comment]
UV was recently updated, so the issue should be resolved (after `--prerelease=allow` is removed).
I reran CI workflows and despite the issue arising at the "Install minimal dependencies" all the tests passed 🤷‍♂️.

Additionally, since now UV seems to work a bit more stable, maybe we should also use it in the GPU workflow?

**Comment by carmocca** [comment]
> Additionally, since now UV seems to work a bit more stable, maybe we should also use it in the GPU workflow?

That would be very welcome Andrei

File: .github/workflows/cpu-tests.yml [hunk 1]
```diff
@@ -46,9 +46,7 @@ jobs:
 
     - name: Install minimal dependencies
       run: |
-        # uv pip install . is not yet supported, only `-e .`
-        # https://github.com/astral-sh/uv/issues/1896
-        uv pip install --system -e .
+        uv pip install --system --prerelease=allow .
```
Comment (carmocca) [inline .github/workflows/cpu-tests.yml:49]:
```suggestion
        uv pip install --system .
```

**Pull request merged**